web/src/app/workplace/components/task-master/task-master.component.ts

```diff
@@ -90,9 +84,10 @@ export class TaskMasterComponent implements OnInit, OnDestroy {
 
     this.workplaceService.objectReferenceSelectedStream.pipe(takeUntil(this.destroy$)).subscribe((objectReference) => {
       this.objectReference = objectReference;
-      delete this.currentBasket;
       if (objectReference) {
         this.getTasks();
+      } else {
+        delete this.currentBasket;
```

Comment:
I thought about this - I think we have to delete the currentBasket even when there is an objectReference. In case we search by type and value, the method getTasks() should be called without a currentBasket. Otherwise the method would filter the tasks by a workbasket (the currentBasket).